Gruntfile.js

```diff
@@ -266,9 +276,32 @@ module.exports = function(grunt) {
         }]
       }
     },
+    less: {
+      development: {
+        options: {
+          compress: false,
+          optimization: 2,
+          strictMath: true
+        },
+        files: [
+          {
+            expand: true,
+            cwd: "",
+            src: ["0-*/**/style/style.less"],
+            dest: "",
+            ext: ".css"
+          }
+        ]
+      }
+    }
   });
 
+  // grunt.registerTask('bake', [
+  //   'less'
+  // ]);
+
   grunt.registerTask('dev', [
+    'less',
```

Comment:
I want `bake` to do its job ***after*** css has been compiled but I can't get it to work :( 

I tried
```
grunt.registerTask('bake', [
  'less'
]);

grunt.registerTask('dev', [
  'bake',
  'imagemin',
  'connect:livereload',
  'watch'
]);

```

But that breaks `bake` completely. Hellllp!